diff --git a/rtl/cve2_core.sv b/rtl/cve2_core.sv
index aff3d0d..82dc163 100644
--- a/rtl/cve2_core.sv
+++ b/rtl/cve2_core.sv
@@ -64,6 +64,7 @@ module cve2_core import cve2_pkg::*; #(
 
   // Debug Interface
   input  logic                         debug_req_i,
+  output logic                         debug_mode_o,
   output crash_dump_t                  crash_dump_o,
   // SEC_CM: EXCEPTION.CTRL_FLOW.LOCAL_ESC
   // SEC_CM: EXCEPTION.CTRL_FLOW.GLOBAL_ESC
@@ -1283,6 +1284,7 @@ module cve2_core import cve2_pkg::*; #(
   logic unused_instr_new_id, unused_instr_id_done;
   assign unused_instr_id_done = instr_id_done;
   assign unused_instr_new_id = instr_new_id;
+  assign debug_mode_o = debug_mode;
 `endif
 
 endmodule
